Explorer/Assets/DCL/ApplicationBlocklistGuard/ApplicationBlocklistGuard.cs

```diff
@@ -13,14 +13,11 @@ public static class ApplicationBlocklistGuard
     {
         public static async UniTask<bool> IsUserBlocklistedAsync(IWebRequestController webRequestController, IDecentralandUrlsSource urlsSource, string userID, CancellationToken ct)
         {
-            FlatFetchResponse response = await webRequestController.GetAsync<FlatFetchResponse<GenericGetRequest>, FlatFetchResponse>(
+            var bd = await webRequestController.GetAsync(
                 urlsSource.Url(DecentralandUrl.Blocklist),
-                new FlatFetchResponse<GenericGetRequest>(),
-                ct,
                 ReportCategory.STARTUP,
-                new WebRequestHeadersInfo());
-
-            BlocklistData bd = JsonUtility.FromJson<BlocklistData>(response.body);
+                new WebRequestHeadersInfo())
+                                               .CreateFromJsonAsync<BlocklistData>(WRJsonParser.Unity, ct);
```

Comment:
Please avoid combining things like this. It is now harder to put a breakpoint between GetAsync and CreateFromJsonAsync.